Documentation/Metrics/arangodb_ioheartbeat_delays_read_total.yaml

```
@@ -0,0 +1,20 @@
+name: arangodb_ioheartbeat_delays_read_total
+introducedIn: "3.8.7"
+help: |
+  Number of delays in the io heartbeat read test.
+unit: number
+type: counter
+category: Health
+complexity: medium
+exposedBy:
+  - dbserver
+  - agent
+  - single
+description: |
+  This counter is increased whenever the io heartbeat encounters a delay
+  of at least 1s when reading a small file in the database directory.
+  This test is done every 15s to ensure that the underlying volume is
```

While 15s may be correct now, I can imagine someone changing the frequency in the code later without updating the metrics description. Shall we change it to "periodically"?
```suggestion
  This test is done periodically to ensure that the underlying volume is
```

---

Done.